### ISSUE TITLE
deepai client inserts API key into every default axios request

### ISSUE THREAD

**Issue by Peter-Maguire** [deepai-org/deepai-js-client#13]
Making an request with axios in the same script that imports deepai will attach the following headers to every request you send, regardless of it's destination:

```
 'client-library: deepai-js-client' 
 'api-key: 8cc49c28-870a-43db-bf17-a84da3b1be55'
```

**Comment by mploux** [comment]
Very strange code. Why don't they create an axios instance ? It is it's intended use.

**Comment by baragona** [comment]
This is fixed in version 1.0.18, thank you.
